pro/cypress/e2e/features/signupJourneyCreateOfferer.feature

```diff
@@ -1,3 +1,29 @@
 @P0
 Feature: Signup journey
-# all tests are currently skipped 'cause flaky, it will be investigated and migrated later
+
+  Scenario: With a new account, create a new offerer with an unknown SIRET
+    Given I log in with a "first" new account
+    When I start offerer creation
+    And I specify an offerer with a SIRET
+    And I add details to offerer
+    And I fill activity step
+    And I validate
+    Then the offerer is created
+
+  Scenario: With a new account and a known offerer, create a new offerer in the space
```

Comment:
in the space? j'ai du mal à voir à quel concept métier ça fait référence

Reply:
espace (FR) -> space (EN) oui je sais, mixer du Français et de l'anglais c'est pas top